roles/simulator/tasks/start.yml

```diff
@@ -1,39 +1,96 @@
 ---
 
-- debug:
-    msg: "{{ ROOT_DIR }}"
-  tags:
-    - ueransim
-
 - name: copy config files to ueransim host
   template:
     src: "{{ ROOT_DIR }}/{{ item.1 }}"
     dest: ueransim_onramp/config/
   with_indexed_items: "{{ ueransim.servers[lookup('ansible.utils.index_of', groups['ueransim_nodes'], 'eq', inventory_hostname)] }}"
   when: inventory_hostname in groups['ueransim_nodes']
-  tags:
-    - ueransim
 
 - name: Run gNB
-  shell: nohup ./nr-gnb -c "../config/{{ item.1 }}" > /dev/null 2>&1 &
+  shell: nohup ./nr-gnb -c "../config/custom-gnb.yaml" > /dev/null 2>&1 &
+  register: gnb_output
+  args:
+    chdir: ueransim_onramp/build
   with_indexed_items: "{{ ueransim.servers[lookup('ansible.utils.index_of', groups['ueransim_nodes'], 'eq', inventory_hostname)] }}"
-  when: "item.0  == 0"
   async: 30
   poll: 0
-  args:
-    chdir: ueransim_onramp/build
-  register: gnb_output
-  tags:
-    - ueransim
+  when: "inventory_hostname in groups['ueransim_nodes'] and item.0  == 0"
 
 - debug:
     var: gnb_output
+  with_indexed_items: "{{ ueransim.servers[lookup('ansible.utils.index_of', groups['ueransim_nodes'], 'eq', inventory_hostname)] }}"
+  when: "inventory_hostname in groups['ueransim_nodes'] and item.0  == 0"  
+
+- pause:
+    seconds: 5
+
 
 - name: Run UE
-  shell: nohup ./nr-ue -c "../config/{{ item.1 }}" > /dev/null 2>&1 &
+  shell: nohup ./nr-ue -c "../config/custom-ue.yaml" > /dev/null 2>&1 &
```

Comment:
Same comment as before...
"I do not think this change is correct because it will work only for a single `ueransim` instance. If the user wants to deploy multiple `ueransim` instances, the same file will be copied to the different instances, which is incorrect. I think this has to be kept as it was before."